src/System.Windows.Forms/src/System/Windows/Forms/ListViewItem.cs

```diff
@@ -246,6 +248,23 @@ public ListViewItem(ListViewSubItem[] subItems, string imageKey, ListViewGroup g
             Group = group;
         }
 
+        internal AccessibleObject AccessibilityObject
+        {
+            get
+            {
+                bool inDefaultGroup =
+                    listView != null && listView.Groups != null && listView.Groups.Count > 0 && Group == null;
```

Comment:
Reorder operands and use null coalescence, e.g.
```suggestion
                bool inDefaultGroup = Group == null && listView?.Groups?.Count > 0;
```

But then there's a Q of why we need to consider `listView?.Groups?.Count` at all?
The property that controls whether groups are shown is [`ShowGroups`](https://docs.microsoft.com/dotnet/api/system.windows.forms.listview.showgroups) and there's also another property that may be relevant `GroupsEnabled`